slatedb/src/db.rs

```diff
@@ -3272,6 +3279,10 @@ mod tests {
         assert!(result.to_string().contains("background task panic'd"));
     }
 
+    // TODO(flaneur2020): it seems that the mem_table_flusher's background task will get the error
```

Comment:
Hm.. I'd like to see this figured out/resolved before merging. Mostly 'cuz this stuff is so critical.